developer_manual/client_apis/WebDAV/chunking.rst

```diff
@@ -10,13 +10,113 @@ Uploading large files is always a bit problematic as your connection can be inte
 which will fail your entire upload. Nextcloud has a chunking API where you can
 upload smaller chunks which will be assembled on the server once they are all uploaded.
 
-Usage
------
+There are two versions of the chunking API. Version 1 is the original version and version 2 was built as a backward compatible extension to support uploads directly to supporting target storages like S3. Version 2 is the recommended version to use.
+
+Version 2 comes with a few additional requirements and limitations to consider (compared to version 1):
+- Every request needs to have a `Destination`` header present which specifies the target path of the file
+- The naming of the individual chunks is limited to be a number between 1 and 10000
+- The chunks will be assembled in the order of their names
+- The size of chunks must be between 5MB and 5GB (except for the last chunk, which can be smaller)
+- Chunks cannot be downloaded from the upload directory
+
+Chunked upload v2
+-----------------
+
+The API is only available for registered users of your instance. And uses the path:
+``<server>/remote.php/dav/uploads/<userid>``. For this guide we will assume:
+``https://server/remote.php/dav/uploads/roeland``
+
+
+
+Starting a chunked upload
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A chunked upload is handled in 1 folder. This is the location all the chunks
+are uploaded to.
+
+Start by creating a folder with a unique name. You can list the current available
+folder but if you take a random UUID chances of collision are tiny.
+
+.. code-block:: console
+
+    curl -X MKCOL -u roeland:pass \
+        https://server/remote.php/dav/uploads/roeland/myapp-e1663913-4423-4efe-a9cd-26e7beeca3c0 \
+        --header 'Destination: Destination
+	https://server/remote.php/dav/files/roeland/dest/file.zip'
```

Comment:
not an error, but the line break is confusing. would expect it ending at the same line, or having further indentation.